### PR TITLE
Changes to create new learner subsection grade API

### PR DESCRIPTION
**Pull request by ziafazal**
Changes to create new learner subsection grade API. This API should provide a get method to list all learner subsection grades for given enterprise.
ENT-2694

### CODE REVIEW

File: enterprise_data/models.py [hunk 1]
```diff
@@ -126,3 +126,49 @@ def __repr__(self):
...
+    subsection_grade_created = models.DateTimeField(null=True)
+    first_attempted = models.DateTimeField(null=True)
+    earned_all = models.FloatField()
+    possible_all = models.FloatField()
+    earned_graded = models.FloatField()
+    possible_graded = models.FloatField()
+
+    class Meta:
+        app_label = 'enterprise_data'
+        db_table = 'enterprise_subsection_grade'
```
Comment (muhammad-ammar) [inline enterprise_data/models.py:164]: @ziafazal I am unable to find this table anywhere?

Comment (ziafazal) [reply enterprise_data/models.py:164]: This is going to be created as result of ENT-2693

File: enterprise_data/filters.py [hunk 2]
```diff
@@ -85,3 +85,17 @@ def filter_queryset(self, request, queryset, view):
             queryset = queryset.exclude(**filter_query)
 
         return queryset
+
+
+class EnterpriseFilterBackend(filters.BaseFilterBackend, FiltersMixin):
```
Comment (hasnain-naveed) [inline enterprise_data/filters.py:90]: Why didn't you do this in view by `get_queryset` function?

Comment (ziafazal) [reply enterprise_data/filters.py:90]: Advantage of Filter is it can be re-used

**Review by hasnain-naveed — APPROVED** [review]
👍

**Pull request closed without merging**